tt_eager/tt_dnn/op_library/moreh_norm/moreh_norm_h/kernels/moreh_norm_h_kernel.cpp

```diff
@@ -0,0 +1,149 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "tt_eager/tt_dnn/op_library/moreh_norm/kernel_utils/common_ckernels.hpp"
+
+namespace NAMESPACE {
+void MAIN {
+    int i{0};
+    const auto num_cols_per_core = get_arg_val<uint32_t>(i++);
+    const auto Ht = get_arg_val<uint32_t>(i++);
+    const auto origin_h = get_arg_val<uint32_t>(i++);
+    const auto p = get_arg_val<uint32_t>(i++);
+    const bool p_is_negative = get_arg_val<uint32_t>(i++) == 1;
+    const auto recip_p = get_arg_val<uint32_t>(i++);
+    const bool recip_p_is_negative = get_arg_val<uint32_t>(i++) == 1;
+
+    std::uint8_t input_id{0};
+    const auto cb_x = input_id++;                // input
+    const auto cb_one = input_id++;              // one
+    const auto cb_decimal = input_id++;          // decimal
+    const auto cb_recip_p_decimal = input_id++;  // recip_p_decimal
+    const auto cb_mask_h = input_id++;           // mask_h
+
+    std::uint8_t output_id{16};
+    const auto cb_y = output_id++;  // output
+
+    std::uint8_t intermed_id{24};
```

Comment:
```suggestion
    std::uint8_t intermed_id{tt::CB::c_intermed0};
```
I think it's better to use enums.

Reply:
> I think it's better to use enums.

Good gomment, thx.